eslint.js

```diff
@@ -1,6 +1,4 @@
-#!/usr/bin/env node
```

Comment:
I removed this by accident.  Before I add this back, does it actually need to be there?

Reply:
Huh - are we ever running it as a binary? If not, then we probably don't need it!